esi_leap/api/controllers/v1/utils.py

```diff
@@ -152,8 +153,14 @@ def offer_get_dict_with_added_info(offer, project_list=None, node_list=None):
 
     o = offer.to_dict()
     o['availabilities'] = availabilities
-    o['project'] = keystone.get_project_name(offer.project_id, project_list)
-    o['lessee'] = keystone.get_project_name(offer.lessee_id, project_list)
+    with concurrent.futures.ThreadPoolExecutor() as executor:
```

Comment:
We actually don't need this to be parallelized; passing in project_list to keystone.get_project_name prevents multiple API calls.